lib/lambda.d/cert-renew-scheduler/package.json

```diff
@@ -17,5 +17,18 @@
     "ts-loader": "^8.0.7",
     "typescript": "^4.0.5"
   },
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "targets": {
```

Comment:
looks like it’s generated by `esbuild` as intermittent config. how about trying again after removing it?

Reply:
Reverted